Sources/Fluent/Model.swift

```diff
@@ -0,0 +1,101 @@
+
+public protocol Model {
+    static var entity: String { get }
+    var id: String? { get }
+    
+    func serialize() -> [String: StatementValueType]
```

Comment:
Currently the only way to set nil is string NULL. I did the optional route and it confuses the StatementValue and wraps it in an unwrap-able optional.